lib/scala/logging-jutil/src/main/java/org/enso/logger/Converter.java

```diff
@@ -0,0 +1,48 @@
+package org.enso.logger;
+
+import static org.slf4j.event.Level.*;
+
+import org.slf4j.event.Level;
+
+public class Converter {
+
+  public static Level defaultLogLevel = TRACE;
+
+  /** Determines what is the smallest Java level that is still debug and not trace. */
+  private static int defaultLevelDebugCutOff =
+      Math.min(java.util.logging.Level.FINE.intValue(), java.util.logging.Level.CONFIG.intValue());
+
+  /**
+   * Converts SLF4J's Level to java.util one.
+   *
+   * @param level the SLF4J's level to convert.
+   * @return an equivalent in java.util.logging.Level terms
+   */
+  public static java.util.logging.Level toJavaLevel(Level level) {
+    switch (level) {
+      case ERROR:
+        return java.util.logging.Level.SEVERE;
+      case WARN:
+        return java.util.logging.Level.WARNING;
+      case INFO:
+        return java.util.logging.Level.INFO;
+      case DEBUG:
+        return java.util.logging.Level.FINE;
+      case TRACE:
+        return java.util.logging.Level.FINEST;
+      default:
+        return java.util.logging.Level.ALL;
```

Comment:
The match is already exhaustive. It should be a dead code, right?

Reply:
We are dealing with Java ;)

Reply:
Write the code like:
```java
    return switch (l) {
      case DEBUG -> -1;
      case ERROR -> -3;
      case INFO -> 5;
      case WARN -> 10;
      case TRACE -> 1;
    };
```
and then you will not need to complain.